lib/imports/api/config/server/prosemeteor-config.js

```diff
@@ -0,0 +1,62 @@
+/**
+* Configuration class for getting and setting config values for ProseMeteor.
+*/
+class ProseMeteorConfig {
```

Comment:
new config class with getters/setters for all config values. the module exports a single instance, making it a singleton